src/ui/components/shared/Icon.tsx

```diff
@@ -1,26 +1,13 @@
-  replay:
-    "M9.741 5.328 4.209 2.107a.784.784 0 0 0-1.078.29.801.801 0 0 0-.106.399v6.443a.801.801 0 0 0 .394.69.784.784 0 0 0 .79 0L9.74 6.707a.792.792 0 0 0 .395-.69.8.8 0 0 0-.395-.689ZM9.716 14.3l-5.532-3.222a.783.783 0 0 0-1.078.291.802.802 0 0 0-.106.398v6.443a.802.802 0 0 0 .395.69.784.784 0 0 0 .789 0l5.532-3.222a.792.792 0 0 0 .395-.69.801.801 0 0 0-.395-.689ZM17.605 9.814l-5.532-3.222a.784.784 0 0 0-1.078.292.8.8 0 0 0-.106.397v6.444a.8.8 0 0 0 .395.689.786.786 0 0 0 .789 0l5.532-3.222a.792.792 0 0 0 .395-.689.802.802 0 0 0-.395-.69Z",
-};
-
-export default function Icon({ iconName }: { iconName: keyof typeof icons }) {
+export default function Icon(props) {
   return (
-    <svg
-      style={{ display: "inline-block", verticalAlign: "middle" }}
-      viewBox={"0 0 20 20"}
-      width={`20px`}
-      height={`20px`}
-      xmlns="http://www.w3.org/2000/svg"
-      xmlnsXlink="http://www.w3.org/1999/xlink"
-    >
-      <path fill={"currentColor"} d={icons[iconName]} />
-    </svg>
+    <img
```

Comment:
Not quite:
1) We already implement something similar to what we're trying to get here so you can copy a good chunk of code (pasting screenshot below for the command palette icon in the editor, but you can inspect that too).
2) In case it's cleaner to just use a proper CSS file here, we should. The way we import is a little bit roundabout. After creating `Icon.css`, make sure you import it in `pages/_app.tsx` along with the other stylesheets.

![image](https://user-images.githubusercontent.com/15959269/152259078-89ed3f62-1cec-4557-9a2e-7f055f7e97ae.png)